### ISSUE TITLE
Please tag a stable release

### ISSUE THREAD

**Issue by sjackman** [bioinfologics/w2rap-contigger#6]
Thanks!

**Comment by bjclavijo** [comment]
Hi Shaun, the master should compile and run OK, we don't have a "stable release" as of now, I am working towards that on my own branch, expect something in the next week or two. It will change the way it is run, etc. I'll leave this open till we create a stable release.

**Comment by sjackman** [comment]
Cool. Thanks, Bernardo.

I have a DISCOVARdenovo assembly which took 8 days. Can I feed those contigs into the w2rap-contigger pipeline?

**Comment by bjclavijo** [comment]
This part is literally the enhancement/replacement for DISCOVAR, so you shouldn't run this on DISCOVAR contigs. If discovar finished in you dataset and produced an assembly, then the results from the w2rap-contigger (barred parameter tweaking) will be quite similar. The rest of the pipeline (i.e. the Nextera LMP scaffolding will be available in the next few weeks.

**Comment by sjackman** [comment]
Ah, okay. Thanks for the clarification. I look forward to the release of the scaffolding pipeline.

**Comment by JingzhX** [comment]
Hi Bernardo. I am not able to compile this software following the installation guide in README file. The OS is Ubuntu server 16.04. Can you tell me how to handle this problem? thanks!

Jingzhong Xie

error message:
CMakeFiles/01_unipaths.dir/src/bam/ReadBAM.cc.o: In function `unzipBlocks':
/sft/src/w2rap-contigger/src/bam/ReadBAM.cc:378: undefined reference to `inflateInit2_'
/sft/src/w2rap-contigger/src/bam/ReadBAM.cc:379: undefined reference to `inflate'
/sft/src/w2rap-contigger/src/bam/ReadBAM.cc:380: undefined reference to `inflateEnd'
CMakeFiles/01_unipaths.dir/src/bam/ReadBAM.cc.o: In function `GZipFooter':
/sft/src/w2rap-contigger/src/bam/ReadBAM.cc:80: undefined reference to `crc32'
CMakeFiles/01_unipaths.dir/src/bam/ReadBAM.cc.o: In function `__static_initialization_and_destruction_0':
/sft/src/w2rap-contigger/src/bam/ReadBAM.cc:91: undefined reference to `crc32'
collect2: error: ld returned 1 exit status
CMakeFiles/01_unipaths.dir/build.make:638: recipe for target 'bin/01_unipaths' failed
make[2]: *** [bin/01_unipaths] Error 1
CMakeFiles/Makefile2:293: recipe for target 'CMakeFiles/01_unipaths.dir/all' failed
make[1]: *** [CMakeFiles/01_unipaths.dir/all] Error 2
make[1]: *** Waiting for unfinished jobs....
[100%] Built target 02_qgraph
[100%] Linking CXX executable bin/05_simplify
[100%] Built target 04_patching
[100%] Built target 06_scaffolding
[100%] Built target 05_simplify
Makefile:83: recipe for target 'all' failed
make: *** [all] Error 2

**Comment by bjclavijo** [comment]
Hi Jingzhong, can you please try the latest commit from today in the master?
If it still doesn't compile, please do open a new issue, as this one is about a stable release.

**Comment by JingzhX** [comment]
It was compiled successfully. Thanks, Bernardo.

**Comment by sjackman** [comment]
Any update on tagging a stable release?

**Comment by bjclavijo** [comment]
Yeah, I was aiming for last week to re-merge my branch back in, which will become the first stable release. I obviously didn't make it for various reasons, but I am trying for next week most likely. So getting very close.
Still a few wrinkles to iron, but definitely getting there.
